docs/docs/getting-started.server.mdx

```diff
@@ -573,40 +573,68 @@ on how to use MDX with Gatsby.
   <summary>Expand example</summary>
 
   ```js path="next.config.js"
-  module.exports = {
-    // Prefer loading of ES Modules over CommonJS
-    experimental: {esmExternals: true},
+  import nextMDX from '@next/mdx'
+
+  const withMDX = nextMDX({
+    // By default only the .mdx extension is supported.
+    extension: /\.mdx?$/,
+    options: {/* providerImportSource: …, otherOptions… */}
+  })
+
+  export default withMDX({
     // Support MDX files as pages:
     pageExtensions: ['md', 'mdx', 'tsx', 'ts', 'jsx', 'js'],
-    // Support loading `.md`, `.mdx`:
-    webpack(config, options) {
-      config.module.rules.push({
-        test: /\.mdx?$/,
-        use: [
-          // The default `babel-loader` used by Next:
-          options.defaultLoaders.babel,
-          {
-            loader: '@mdx-js/loader',
-            /** @type {import('@mdx-js/loader').Options} */
-            options: {/* jsxImportSource: …, otherOptions… */}
-          }
-        ]
-      })
-
-      return config
-    }
-  }
+  })
   ```
 </details>
 
-[Next](https://nextjs.org) supports webpack loaders by overwriting the webpack
-config in `next.config.js`.
+[Next.js](https://nextjs.org) have their own package for supporting MDX.
 
-Install and configure the webpack loader [`@mdx-js/loader`][webpack].
+Install and configure [`@next/mdx`][@next/mdx].
 There is no need to configure your JSX runtime as React is already set up.
 
-See also [¶ Webpack][webpack] and [¶ React][react], which you’re using, for
-more on those tools.
+The React provider can be configured in [`pages/_app.js`](https://nextjs.org/docs/advanced-features/custom-app).
```

Comment:
Reference link w/ definition at bottom is the preference for most links

Reply:
I don’t think Next can work with Preact or something else, right?
Should `@mdx-js/react` be installed?
Should it be called React provider or MDX provider? MDX provider for React?

Reply:
Afaik Next only works with React. I think naming it `MDX provider` makes sense.